lib/mirror.js

```diff
@@ -480,8 +496,8 @@ class Mirror {
                     folderDirParts.pop();
                     Mirror.createRecursiveDir(folderDirParts.join('/'));
```

Comment:
If createRecursiveDir fails, should we even continue here?

Reply:
We probably should not. Not sure how to best change the situation.

Reply:
Ok, had an idea... hope you like it, see e7412ba.